### ISSUE TITLE
Create a separate component for mentor card (horizontal view)

### ISSUE THREAD

**Issue by Piumal1999** [sef-global/scholarx-frontend#105]
**Is your feature request related to a problem? Please describe.**
In the admin's mentee profile view, we are currently displaying the mentor details like this:
<img width="462" alt="image" src="https://github.com/sef-global/scholarx-frontend/assets/27498587/80600331-4d25-4dcf-8a03-9bea94db0dcb">

Currently, it is built inside the MentorProfile component, rather than as a separate component.

**Describe the solution you'd like**
Make it a separate component. Consider other pages where this component can be reused and adjust the design accordingly.

**Describe alternatives you've considered**
N/A

**Additional context**
Bug introduced with #103

**Comment by dilankavishka** [comment]
Shall i work on this issue please? @Piumal1999

**Comment by anjula-sack** [comment]
Any updates on this? @dilankavishka

**Comment by dilankavishka** [comment]
can i get some more time to do this work please? @anjula-sack

**Comment by anjula-sack** [comment]
Any updates @dilankavishka

**Comment by dilankavishka** [comment]
Yeah anjula. i'm working with this.. maybe i am able do finish this before this saturday

**Comment by anjula-sack** [comment]
Are you working on this? @dilankavishka

**Comment by dilankavishka** [comment]
Yeah.
